crates/bin/prove_block/src/reexecute.rs

```diff
@@ -147,8 +147,11 @@ pub(crate) fn format_commitment_facts<H: HashFunctionType>(
 impl PerContractStorage for ProverPerContractStorage {
     async fn compute_commitment(&mut self) -> Result<CommitmentInfo, CommitmentInfoError> {
         // TODO: error code
```

Comment:
@ftheirs please add a comment that explain the motivation of this fix. 
In general storage_proofs **needs** to have this field.

Reply:
Specifically which contracts are and why the start having contract_data in block 10

Reply:
In pathfinder, the structure from the requested type is defined with Opt<ContractData>:
https://github.com/Moonsong-Labs/pathfinder/blob/c45b23d2d6407079ad19f84ef0b23f913063d7cd/crates/rpc/src/pathfinder/methods/get_proof.rs#L182-L199
Besides, when we work with a devnet, we're returning in an early part of the function. We get a None when we retrieve the `contract_state_hash` from the DB, and this leads to the early return where most of the fields are empty/none
https://github.com/Moonsong-Labs/pathfinder/blob/c45b23d2d6407079ad19f84ef0b23f913063d7cd/crates/rpc/src/pathfinder/methods/get_proof.rs#L283-L294

When we bump into this condition, the fix just create a default ContractData structure with all fields in zero